examples/tagmap/app.js

```diff
@@ -78,7 +78,7 @@ class Root extends Component {
 
     requestJson(FILE_PATH, (error, response) => {
```

Comment:
For another PR: we should update examples to use `fetch` so we can drop another dependency...